mesonbuild/environment.py

```diff
@@ -805,7 +807,7 @@ def _set_default_options_from_env(self) -> None:
                     p_list = list(mesonlib.OrderedSet(p_env.split(':')))
                 else:
                     p_list = split_args(p_env)
-                p_list = [e for e in p_list if e]  # filter out any empty eelemnts
+                p_list = [e for e in p_list if e]  # filter out any empty elemnts
```

Comment:
Awkward, but... `elements`.

Reply:
gahh, my spell checker has decided that it will *not* not matter what I tell it check .py files anymore :cry: